### PR TITLE
ribbon styling updates

### PR DESCRIPTION
**Pull request by SidelineCory24**

### CODE REVIEW

**Comment by SidelineCory24** [comment]
These are the updates based off of Chris's feedback. I still need to meet with him for the final okay.

**Comment by SidelineCory24** [comment]
Updates:
1. Removed featured ribbon
2. Added StaticDev to each of the new ribbons (CN, HTF, SND)
3. Added "dummydata" to Bureau Results Card due to styling issue caused when adding StaticDev instead

**Comment by SidelineCory24** [comment]
@elizabeth-jimenez should be gtg pending the checks

**Comment by elizabeth-jimenez** [comment]
@SidelineCory24 would removing Featured ribbon apply to the whole application, since the new ribbons are slated to be its replacement?

File: src/Components/BureauPage/BureauResultsCard/BureauResultsCard.jsx [hunk 1]
```diff
@@ -88,14 +88,14 @@ class BureauResultsCard extends Component {
...
+            {
+              get(result, 'dummydata', false) && <CriticalNeed isWide cutSide="both" className="ribbon-results-card" />
             }
             {
-              get(result, 'isDifficultToStaff', false) && <HardToFill isWide cutSide="both" className="ribbon-results-card" />
+              get(result, 'dummydata', false) && <HardToFill isWide cutSide="both" className="ribbon-results-card" />
             }
             {
-              get(result, 'isServiceNeedDifferential', false) && <ServiceNeedDifferential isWide cutSide="both" className="ribbon-results-card" />
+              get(result, 'dummydata', false) && <ServiceNeedDifferential isWide cutSide="both" className="ribbon-results-card" />
```
Comment (elizabeth-jimenez) [inline src/Components/BureauPage/BureauResultsCard/BureauResultsCard.jsx:98]: What if we change `dummydata` to `staticDevContentAlt`? That way we are more likely to remember why we didn't wrap this in Static Dev Content

File: src/Components/ResultsCard/ResultsCard.jsx [hunk 2]
```diff
@@ -303,25 +303,28 @@ class ResultsCard extends Component {
...
                       </StaticDevContent>
                     }
                     {
-                      get(result, 'isDifficultToStaff', false) && <HardToFill className="ribbon-results-card" />
-                    }
-                    {
-                      get(result, 'isServiceNeedDifferential', false) && <ServiceNeedDifferential className="ribbon-results-card" />
+                      get(result, 'isDifficultToStaff', false) &&
+                      <StaticDevContent>
+                        <HardToFill isWideResults className="ribbon-results-card" />
```
Comment (elizabeth-jimenez) [inline src/Components/ResultsCard/ResultsCard.jsx:316]: `isWideResults` doesn't change anything when added here, but if we ever update the name, everything else will be ready to go

Comment (elizabeth-jimenez) [reply src/Components/ResultsCard/ResultsCard.jsx:316]: annnd added this comment in the wrong place, this is for SND on line 322 😅

File: src/Components/BureauPage/BureauResultsCard/BureauResultsCard.jsx (hunk 1 above)
Comment (SidelineCory24) [reply src/Components/BureauPage/BureauResultsCard/BureauResultsCard.jsx:98]: done

File: src/Components/ResultsCard/ResultsCard.jsx (hunk 2 above)
Comment (SidelineCory24) [reply src/Components/ResultsCard/ResultsCard.jsx:316]: done

File: src/Components/Ribbon/Ribbon.test.jsx [hunk 3]
```diff
@@ -28,7 +28,7 @@ describe('RibbonComponent', () => {
     const wrapper = shallow(<Ribbon {...props$} />);
 
     expect(wrapper.find('div').at(0).props().className)
-      .toBe(`ribbon-outer-container-cut-${props$.cutSide}  ${props$.className}`);
+      .toBe(`ribbon-outer-container-cut-${props$.cutSide}   ${props$.className}`);
```
Comment (elizabeth-jimenez) [inline src/Components/Ribbon/Ribbon.test.jsx:31]: nice catch 🕵️‍♂️

**Review by elizabeth-jimenez — APPROVED** [review]
🍰 🧁 🥮 👍

**Pull request merged**